reflex/event.py

```diff
@@ -330,6 +330,29 @@ def set_clipboard(content: str) -> EventSpec:
     )
 
 
+def download(url: str, filename: str) -> EventSpec:
```

Comment:
If they don't provide a filename, maybe we can default to the last part of the path in the url? Basically give it the same filename it was saved as

Reply:
Sure we could do that